### PR TITLE
Disable new no-type-only-packages rule in packages which fail

### PR DESCRIPTION
**Pull request by jakebailey**
https://github.com/microsoft/DefinitelyTyped-tools/pull/865 adds a new lint rule which errors when a package does not contain any declarations in value space (or augment other packages/globals) and therefore do not describe actual JS code that can be imported or referenced.

We prefer that these kinds of packages are not added to the repo without some sort of clear rationale, per https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-type-definitions-with-no-matching-package.

Only 100 packages fail this rule today; this PR exempts them.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "@definitelytyped/no-type-only-packages": "off"
```
Comment (sheetalkamat) [inline .eslintrc.json:3]: why do we need this at root if we are adding it to individual packages

Comment (jakebailey) [reply .eslintrc.json:3]: Oops, that is a typo in my script.

Comment (jakebailey) [reply .eslintrc.json]: Thanks for catching this; my grepping netted an empty string result and so that joined with the repo root in the script!

**Review by sheetalkamat — APPROVED** [review]

**Comment by jakebailey** [comment]
Will merge once the DT-tools PR is in and released (and I've had a chance to double check performance)

**Pull request merged**